release/models/bgp/openconfig-bgp-policy.yang

```diff
@@ -796,6 +807,449 @@ module openconfig-bgp-policy {
     }
   }
 
+  grouping community-members-top {
+    description
+      "Top-level grouping for members of a standard community set";
+
+    container members {
+        description
+            "Enclosing container for list of standard community members";
+
+        list member {
+            key "index";
+            description
+              "List of members in an extended community set.  Each entry
+              is identified by an index.";
+
+          leaf index {
+              type leafref {
+                  path "../config/index";
+              }
+              description
+                  "Reference to the index for the member of an standard
+                  community set.";
+          }
+
+          container config {
+            description "Standard community member configuration.";
+
+            uses community-member-config;
+          }
+
+          container state {
+            config false;
+            description "State for BGP standard community set members";
+
+            uses community-member-config;
+            uses community-member-state;
+          }
+
+          container asn-type {
+            description
+            "RFC4360 section 3.1 BGP community with 2 byte ASN.";
+
+            container config {
+              description "Config for asn-type extended community.";
+              uses community-asn-type-config;
+            }
+            container state {
+              description "State for asn-type extended community.";
+              config false;
+              uses community-asn-type-config;
+            }
+          }
+
+          container asn-regex-type {
+            description
+            "A regex used to match RFC4360 section 3.1 BGP community with 2 byte ASN.";
+
+            container config {
+              description "Config for asn-regex-type community.";
+              uses community-regex-type-config;
+            }
+            container state {
+              description "State for asn-regex-type community.";
+              config false;
+              uses community-regex-type-config;
+            }
+          }
+
+        }
+    }
+  }
+
+  grouping community-member-config {
+    description
+    "Configuration data for BGP standard communities.";
+
+    leaf index {
+      type uint16;
+      description
+        "Represents an index of a member entry.";
+    }
```

Comment:
This needs to have a community value.
There should also be a community regex option (list of regexes)